test/unittest/test_tarantool_server.py

```diff
@@ -0,0 +1,43 @@
+import os
+import subprocess
+import unittest
+
+
+class TestTarantoolServer(unittest.TestCase):
+    def test_tarantool_server_not_hanging(self):
+        env = os.environ.copy()
+        env['SERVER_START_TIMEOUT'] = '5'
+
+        cmd = ['python', 'test/test-run.py', 'unittest/hang.test.lua']
```

Comment:
We use python3 in shebangs, because there are distros (say, CentOS 7), where python is alias for python2. Here we can use `sys.executable` to run test-run using exactly same interpreter as one, which runs the testing.